src/main/java/com/microsoft/sqlserver/jdbc/Geography.java

```diff
@@ -15,6 +15,8 @@
 
 public class Geography extends SQLServerSpatialDatatype {
 
+    protected Geography() {}
+
     /**
      * Private constructor used for creating a Geography object from WKT and Spatial Reference Identifier.
```

Comment:
protected

Reply:
Protected is fine - came from other issue, but is this particular constructor needed now?

Reply:
Yes, if we don't specify protected here, the default constructor will have package private visibility, and it won't let other classes outside the package inherit this class. Therefore, we need this line.